lib/protocol_buffers/runtime/service.rb

```
@@ -1,10 +1,60 @@
 require 'protocol_buffers/runtime/rpc'
+require 'protocol_buffers/compiler/file_descriptor_to_ruby'
+require 'protocol_buffers/compiler/fully_qualified_name'
+
+module ProtocolBuffers
+  class ServiceRegistry
+    def self.register_service(fully_qualified_name, handler_factory)
+      return nil unless fully_qualified_name && handler_factory
```

silent error on fully_qualified_name